docs/manage-metalake-using-gravitino.md

```diff
@@ -107,15 +109,10 @@ curl -X PUT -H "Accept: application/vnd.gravitino.v1+json" \
   "updates": [
     {
       "@type": "rename",
-      "newName": "metalake"
-    },
-    {
-      "@type": "setProperty",
-      "property": "key2",
-      "value": "value2"
```

Comment:
Any reason removing these lines?

Reply:
As described in the introductory sentence, the example is designed to focus on one of the alterations -- renaming a metalake. That's why I intended to remove these lines. Does it make sense?

Reply:
OK, I see. I think it makes sense.

Reply:
“altering a metalake” here may include multiple changes, e.g, alter the name and set a property to the metadata. So I think the original version is okay, which let user know this API supports multiple changes in one request. What do you think?

Reply:
@shaofengshi thanks for your review comments. Let me explain my intention a bit more:
1. I agree that alteration implies multiple types of changes. However, the original [Java example](https://github.com/apache/gravitino/pull/4997/files#diff-6c143eb15fabcf14ab059a759f44c8c93dea1c8cf4c3f033d9cadb0809cd97e6R123) only introduces the renaming case, which brings inconsistencies among languages.
2. A [complete table](https://github.com/apache/gravitino/pull/4997/files#diff-6c143eb15fabcf14ab059a759f44c8c93dea1c8cf4c3f033d9cadb0809cd97e6R145) down below outlines the allowed changes, which would be more efficient for users to get to know the whole picture of altering. Duplicating them in examples is not necessary, because examples are intended to be as simple as possible -- especially the altering scope may scale as Gravitino evolves.